appveyor.yml

```diff
@@ -17,6 +17,9 @@ environment:
     - ruby_version: "21" # Older version, but matches Travis-CI
     - ruby_version: "21-x64"
 
+init:
+  - git config --global core.autocrlf true
```

Comment:
This is better handled by creating a `.gitattributes` file that specifies which files should be done in what mode. For example to set all files as text mode with `LF` line endings regardless of the platform you would do:
```
* text eol=lf
```